src/test/java/uk/gov/hmcts/reform/sscs/service/LetterUtilsTest.java

```diff
@@ -415,4 +430,250 @@ private String getExpectedName(final String otherPartyId, final NotificationWrap
                 .orElse(""));
     }
 
+    private SscsCaseData setupTestData(DynamicList sender) {
+        return SscsCaseData.builder()
+                .originalSender(sender)
+                .jointParty(JOINT_PARTY_WITH_ID)
+                .otherParties(buildOtherPartyData())
+                .appeal(Appeal.builder()
+                    .appellant(APPELLANT_WITH_ID)
+                    .rep(REPRESENTATIVE_WITH_ID)
+                    .build())
+                .build();
+    }
```

Comment:
would prefer this to be a global variable caseData initialised in @BeforeEach. Then for each test, you can do caseData.setOriginalSender(sender).

Reply:
The setup data is required only for few test case. It is not required for all the test case.